src/mongo/commands.ts

```diff
@@ -105,7 +105,10 @@ export class MongoScriptDocumentVisitor extends MongoVisitor<MongoCommand[]> {
 		if (argumentsContext) {
 			let functionCallContext = argumentsContext.parent;
 			if (functionCallContext && functionCallContext.parent instanceof mongoParser.CommandContext) {
-				this.commands[this.commands.length - 1].arguments = ctx.text;
+				if (!this.commands[this.commands.length - 1].arguments) {
```

Comment:
extract the expression this.commands[this.commands.length - 1], you're using it three times